src/Features/CSharp/Portable/CSharpFeaturesResources.resx

```diff
@@ -569,4 +569,7 @@
   <data name="Remove_unused_variables" xml:space="preserve">
     <value>Remove unused variables</value>
   </data>
+  <data name="Use_default_expression" xml:space="preserve">
+    <value>Use default expression</value>
```

Comment:
I'm not sure what a proper name for this would be. "Use default expression" sounds good as the opposite of "Use default literal", but the problem is that UseDefaultLiteral actually has a different string: "Simplify default expression". That combination doesn't sound right because UseDefaultLiteral suggest it is only simplifying the default expression, but the result will still be a default expression. Then this feature might suggest to use a default expression, as if the current form isn't a default expression. But I can't think of any good opposite of "Simplify default expression". Maybe "expand default expression"? I'm not really fan of that.

Reply:
Why not just print out `Use 'default(int)'`?

Reply:
I'm going to have to also use the simplifier on the document before I can show the type. Is that OK?

Reply:
What about "Use explicit default expression"?

Reply:
> Use default expression

This seems fine to me.

Reply:
> I'm going to have to also use the simplifier on the document before I can show the type. Is that OK?

You just need to get the SymbolDisplay.  It's fine if this said `default(SomeType)` but ended up emitting `default(SomeNamespace.SomeType)`.  The goal is to convey the general intent here :)   Note: there is *ample* precedent for this among many analyzers/fixers.  We common show signatures and whatnot in the lightbulb, but we're ok with SymbolDisplay giving a reasonable non-confusing piece of text to show the user.

--

Also, if this itself is getting too complex, i don't mind "Use default expression".  My suggestion was just an idea, if it was feasible to do :)   I think SymbolDisplay would be simple and sufficient.  But it's up to you :)

Reply:
➡️ Switched to using SymbolDisplay in 4cce95cd76027e316959275779b63a4261358113